pcdsdevices/state.py

```diff
@@ -406,7 +404,8 @@ def subscribe(self, cb, event_type=None, run=True):
         cid = super().subscribe(cb, event_type=event_type, run=run)
         if (event_type == self.SUB_READBACK and not
                 self._has_subscribed_readback):
-            self.readback.subscribe(self._run_sub_readback, run=False)
+            self.motor.user_readback.subscribe(self._run_sub_readback,
```

Comment:
I could see the benefit of a more targeted error message suggesting the possibility of an IOC upgrade if this times out. On the other hand, lots of things could cause `TimeoutError`s...

Reply:
I like this idea for our transition phase where we haven't updated everything. If the motor times out but nothing else does, then surely that means we didn't update.